README.md

```diff
@@ -117,16 +117,20 @@ sh 1.run_preprocessing.sh
 ```
 **Slurm**:
 ```sh
-sh 1.run_preprocessing_slurm.sh
+sbatch 1.run_preprocessing_slurm.sh
 ```
 
+***Note**: For testing purposes, users may only have generated sbs or phenotype images.
+If this is the case, and one of the `SBS_SAMPLES_DF_FP`/`PHENOTYPE_SAMPLES_DF_FP` are empty, then this will impede any further analysis of the missing files.
```

Comment:
Is this accurate? I noticed you changed the if statements for sbs and phenotype in the snakefile, but will things also be ok at preprocessing?

Only comment I have before merging is to ensure that it is very clear how the analysis runs if 1) you are missing one data type or 2) you want to just one 1/2 analyses